java/common/exception/ErrorMessage.java

```diff
@@ -79,14 +79,12 @@ public class ErrorMessage extends com.vaticle.typedb.common.exception.ErrorMessa
             new ErrorMessage(28, "Rule '%s' 'when' has not been provided with any patterns.");
     public static final ErrorMessage INVALID_RULE_WHEN_NESTED_NEGATION =
             new ErrorMessage(29, "Rule '%s' 'when' contains a nested negation.");
-    public static final ErrorMessage INVALID_RULE_WHEN_CONTAINS_DISJUNCTION =
-            new ErrorMessage(30, "Rule '%s' 'when' contains a disjunction.");
     public static final ErrorMessage INVALID_RULE_THEN =
```

Comment:
we currently have the habit of bumping down all the version numbers so there are no gaps

Reply:
Updated.
But we should fix that with the rust version, at least.